specification/common-types/resource-management/v5/networksecurityperimeter.json

```diff
@@ -333,8 +354,7 @@
           "items": {
             "type": "string",
             "format": "arm-id",
-            "description": "Fully qualified resource ID of the suggested resource",
-            "readOnly": true
+            "description": "A type definition that refers the id to an Azure Resource Manager resource."
```

Comment:
>A type definition that refers the id to an Azure Resource Manager resource. [](http://example.com/codeflow?start=28&length=75)

this new description doesn't make sense to me. Isn't the old one better?
Then again, does this items thing support descriptions?? We seem to not usually have a description here.

Reply:
I did some research about supporting the description and it does support it, https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#fixed-fields-7 , for items it is just expecting an schema object  and the schema object supports the description: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#fixed-fields-7